Dockerfile

```diff
@@ -86,16 +82,31 @@ RUN cp -R /tmp/koel/. /var/www/html
 RUN [ ! -f /var/www/html/public/manifest.json ] && cp /var/www/html/public/manifest.json.example /var/www/html/public/manifest.json || true
 RUN chown -R www-data:www-data /var/www/html
 
-# Create /tmp/koel if it doesn't exist, and set ownership to www-data
-RUN mkdir -p /tmp/koel \
- && chown www-data:www-data /tmp/koel \
- && chmod 755 /tmp/koel
+RUN mkdir /music \
+  && chown www-data:www-data /music \
+  && mkdir -p /images/artists \
+  && mkdir -p /images/avatars \
+  && mkdir -p /images/covers \
+  && mkdir -p /images/playlists \
+  && mkdir -p /images/radio-stations \
+  && chown -R www-data:www-data /images \
+  && chmod -R 755 /images \
+  # redirect public img storage into the cache, putting this here and not with the other koel file setups because it makes more sense here, and they are empty folder so don't have a significant impact on file size
+  && rm -r /var/www/html/public/img/artists \
+  && rm -r /var/www/html/public/img/avatars \
+  && rm -r /var/www/html/public/img/covers \
+  && rm -r /var/www/html/public/img/playlists \
```

Comment:
You're missing `radio-stations` here I'd assume.

Reply:
right, added that one in after and missed it here. thanks